### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by gundalow**
More CI fixes
(and remove trailing whietspace

### CODE REVIEW

**Comment by gundalow** [comment]
@qaxi Could you help me understand what these GitHub Action workflow files are for:
ansible-manual-sanity-tests.yml
ansible-manual-unit-tests.yml
ansible-collection-tests.yml
ansible-tests-fast.yml


Seems like we have some duplicated testing.

**Comment by qaxi** [comment]
@gundalow 

> ansible-manual-sanity-tests.yml
ansible-manual-unit-tests.yml
ansible-tests-fast.yml
ansible-test.yml

Should be deleted, they are "by hand" tests - obsoleted by "ansible-collection-tests.yml" - standard test from  ansible-collections /
collection_template 

The only thing I am unsure is "ansible-galaxy-deploy.yml" for deploying to Galaxy, but I suppose "Add Zuul to publish collection" in task list will solve my concerns .

**Comment by qaxi** [comment]
Working on solving  this problem
>       super().setUp()
E       TypeError: super() takes at least 1 argument (0 given)

`super(TestCiscoSMBCommandModule,self).setUp()` should work OK

**Comment by gundalow** [comment]
@qaxi Which versions of Python does this collection support?

**Comment by gundalow** [comment]
@qaxi As of `9af1f16e3ff1b977f24817c388e39f2861cc289b`  `ansible-test units -v --color  --coverage --docker --python 3.8` passes

**Comment by qaxi** [comment]
> @qaxi Which versions of Python does this collection support?

I would like to drop python 2.6 if it is allowed ... but the tests from ansible-test --docker test it from 2.6 to 3.9 so I should pass them ...

**Comment by qaxi** [comment]
I am working on Py 2.6 compatibility in branch https://github.com/ansible-collections/community.ciscosmb/tree/fix-ci-2.6

I solved `super()`, but now I am stuck at
```
self.facts["spacetotal_mb"] = round(int(match[2]) / 1024, 1)
E               TypeError: '_sre.SRE_Match' object is unsubscriptable
```

it looks like this
`self.facts["spacetotal_mb"] = round(int(match.group(2)) / 1024, 1)`
could be solution but ... what pops up later ... who knows ...

@gundalow , whats guideline for backward compatibility (I read something about RHEL 6 and Py2.6) and is it mandatory, recommended ...?

**Comment by tadeboro** [comment]
> whats guideline for backward compatibility (I read something about RHEL 6 and Py2.6) and is it mandatory, recommended ...?

Python 2.6 is a nice-to-have thing. See https://github.com/ansible-community/community-topics/issues/6 for a bit more info.

**Comment by gundalow** [comment]
Felix said:
> match.group(2) looks safer to me, in fact I didn't knew about the [] syntax, I guess that's "newer"

**Comment by gundalow** [comment]
@qaxi I've disabled the Python 2.6 tests.
 Looks like Python 2.7 also fails, see https://github.com/ansible-collections/community.ciscosmb/pull/27/checks

**Comment by qaxi** [comment]
OK, I am triing to solve Py 2.6 in https://github.com/ansible-collections/community.ciscosmb/tree/fix-ci-2.6

still mixed success ...

**Comment by qaxi** [comment]
Ha ....
https://github.com/ansible-collections/community.ciscosmb/runs/2346276809?check_suite_focus=true suceed !!!

Except comment block, I will correct it and merge it to this pull-request soon ...

**Comment by qaxi** [comment]
I think we are done.

I will try enable 2.6 again and will see, do you agree ???

**Comment by qaxi** [comment]
Wow ... all green.

File: tests/unit/plugins/modules/ciscosmb/test_ciscosmb_facts-SG350-28-K9.py [hunk 1]
```diff
@@ -99,11 +99,10 @@ def test_ciscosmb_facts_default(self):
             result['ansible_facts']['ansible_net_serialnum'], 'ABC1234567A'
         )
 
-        with self.assertRaises(KeyError) as cm:
-            len(result['ansible_facts']['ansible_net_stacked_models'])
-        the_exception = cm.exception
-        print(the_exception)
-        self.assertIsInstance(the_exception, KeyError)
+# TODO         with self.assertRaises(KeyError) as cm:
```
Comment (gundalow) [inline tests/unit/plugins/modules/ciscosmb/test_ciscosmb_facts-SG350-28-K9.py:102]: Do you want to raise an issue to track this?

Comment (qaxi) [reply tests/unit/plugins/modules/ciscosmb/test_ciscosmb_facts-SG350-28-K9.py:102]: No, I will work no unit tests later.

Currently there are full units only for one type of devices SG550.

**Comment by qaxi** [comment]
@gundalow Thank you, I would not be able get trough CI without your expertise

**Comment by gundalow** [comment]
@qaxi You are most welcome. Initial setup always requires some work.

**Pull request merged**